src/components/Card/card.tsx

```diff
@@ -0,0 +1,42 @@
+import style from "./card.module.scss";
+import starOn from "../../../public/images/star-on.svg";
+import starOff from "../../../public/images/star-off.svg";
+import MoreSee from "../../../public/images/threeMoreDots.svg";
+import logo from "../../../public/images/logo.svg";
+import {useState} from "react";
+import Image from "next/image";
+
+function Card({}) {
+  const [toggleStar, setToggleStar] = useState(true);
+  function handleStar() {
+    setToggleStar(!toggleStar);
+  }
```

Comment:
이전 상태값을 이용해서 새로운 상태값을 만드는 경우에는, 콜백 함수로 값을 업데이트 시켜주는게 좋습니다~
```suggestion
  function handleStar() {
    setToggleStar(prevToggleStar => !prevToggleStar);
  }
```